modules/acm_sku/config/schema/acm_sku.schema.yml

```diff
@@ -29,9 +29,21 @@ acm_sku.configurable_form_settings:
       sequence:
         type: mapping
         mapping:
-          size:
-            label: 'Size'
-            type: integer
+          attribute_set:
```

Comment:
this schema change was required to make sure tests work fine for dynamically added fields

```
Schema errors for acm_sku.configurable_form_settings with the following errors: acm_sku.configurable_form_settings:attribute_weights.default.color missing schema
```